game/lobby_test.go

```diff
@@ -28,3 +28,22 @@ func Test_CalculateVotesNeededToKick(t *testing.T) {
 		}
 	})
 }
+
+func Test_RemoveAccents(t *testing.T) {
+	t.Run("Check removing accented characters", func(test *testing.T) {
+		var expectedResults = map[string]string{
```

Comment:
While it is good to have some basic tests, a unit-test should cover both sane cases and insane cases. For example, what happens if you give the function an empty string, or a string with just spaces, or very weird characters that are at the very far end of the unicode range. Or even newline characterss.

Reply:
Yup, first time I write/use unit tests too, so I was not sure how it was supposed to work.